data/mi/municipalities/Sarah-H-McClure-2febfd27-8ac5-454a-b758-6bccc53126c0.yml

```diff
@@ -0,0 +1,15 @@
+id: ocd-person/2febfd27-8ac5-454a-b758-6bccc53126c0
+name: Sarah H. McClure
```

Comment:
Looks like there's a duplicate of her, one for West Bloomfield and the other for Bloomfield Hills. Was that intentional?

Reply:
will have to investigate in airtable/git diff

Reply:
@csnardi  yeah, was a duplicate. I deleted the other one